styles.css

```diff
@@ -0,0 +1,40 @@
+.column {
+    float: left;
+    width: 181px;
+    height: 300px;
+    padding: 0px 5px 0px 10px;
+}
+
+#article-border-right {
+    border-right: 1px solid #aba2a2;
+
+}
+
+figure {
+    margin-block-start: 0em;
+    margin-block-end: 0em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
```

Comment:
you dont have to use inline-block etc.. you can just use margin and set the display to either block or inline